lib/chef/provider/package/windows.rb

```diff
@@ -201,6 +201,7 @@ def downloadable_file_missing?
         def resource_for_provider
           @resource_for_provider = Chef::Resource::WindowsPackage.new(new_resource.name).tap do |r|
             r.source(Chef::Util::PathHelper.validate_path(source_location)) unless source_location.nil?
+            r.cookbook_name = new_resource.cookbook_name
```

Comment:
this should match the existing style, i.e. `r.cookbook_name(new_resource.cookbook_name)`